k8s/configmaps.go

```diff
@@ -36,6 +38,16 @@ func ListConfigMapsNamesIfConnected(ctx context.Context, namespaceOverride strin
 				return []string{}, nil
 			}
 		}
+
+		// connection refused
+		if errors.Is(err, syscall.ECONNREFUSED) {
+			return []string{}, nil
+		}
+
+		// invalid configuration: no configuration has been provided
+		if strings.HasPrefix(err.Error(), "invalid configuration") {
```

Comment:
I believe there some `somepkg.Is*()` checks for this

Reply:
I tried to find it, but hasn't been successful.

Reply:
Okay, this one might be it: https://pkg.go.dev/k8s.io/client-go/tools/clientcmd#IsEmptyConfig